source/ranges.tex

```diff
@@ -6096,6 +6145,303 @@
 \tcode{ranges::iter_swap(x.\exposid{i_}.\placeholdernc{current}, y.\exposid{i_}.\placeholdernc{current})}.
 \end{itemdescr}
 
+\rSec2[range.split]{Split view}
+
+\rSec3[range.split.overview]{Overview}
+
+\pnum
+\tcode{split_view} takes a \libconcept{view} and a delimiter, and
+splits the \libconcept{view} into \tcode{subrange}s on the delimiter.
+The delimiter can be a single element or a \libconcept{view} of elements.
+
+\pnum
+The name \tcode{views::split} denotes
+a range adaptor object\iref{range.adaptor.object}.
+Given subexpressions \tcode{E} and \tcode{F},
+the expression \tcode{views::split(E, F)} is expression-equivalent to
+\tcode{split_view(E, F)}.
+
+\pnum
+\begin{example}
+\begin{codeblock}
+string str{"the quick brown fox"};
+for (span<char const> word : split(str, ' ')) {
+  for (char ch : word)
+    cout << ch;
+  cout << '*';
+}
+// The above prints: the*quick*brown*fox*
+\end{codeblock}
+\end{example}
+
+\rSec3[range.split.view]{Class template \tcode{split_view}}
+
+\begin{codeblock}
+namespace std::ranges {
+  template<@\libconcept{forward_range}@ V, @\libconcept{forward_range}@ Pattern>
+    requires @\libconcept{view}@<V> && @\libconcept{view}@<Pattern> &&
+             @\libconcept{indirectly_comparable}@<iterator_t<V>, iterator_t<Pattern>, ranges::equal_to>
+  class split_view : public view_interface<split_view<V, Pattern>> {
+  private:
+    V @\exposid{base_}@ = V();                              // \expos
+    Pattern @\exposid{pattern_}@ = Pattern();               // \expos
+    // \ref{range.split.iterator}, class \tcode{split_view::\exposid{iterator}}
+    struct @\exposid{iterator}@;                            // \expos
+    // \ref{range.split.sentinel}, class \tcode{split_view::\exposid{sentinel}}
+    struct @\exposid{sentinel}@;                            // \expos
+  public:
+    split_view() = default;
```

Comment:
P2325R3 says that "the same kind of change" (i.e., adding `requires default_initializable<V> && default_initializable<Pattern>`) needs to be applied here.

Reply:
Yes, I noticed, thanks! Will apply this.